chore: wip getting started (#7)
kubeshop/testkube-docs: docs/articles/tutorial/1-introduction.md

# Introduction

Welcome to Testkube!

Testkube is a test orchestration and execution framework.
You can automate both simple and complex multi-tool testing scenarios and analyse all of your setup in one centralised place.

## Key Features

#### Test Workflows within your own Kubernetes clusters

> Testkube run tests right from within Kubernetes clusters located on your own premise.

Reviewer: ```suggestion
Testkube runs tests right from within Kubernetes clusters located on your own premise.
```

You can reduce the gap between testing and production environments and

> leverage the Kubernetes scheduler to run tests at large scale.

Reviewer: ```suggestion
leverage the Kubernetes scheduler to run tests on a large scale.
```

Test Workflows make it simple to orchestrate both simple and complex tests.
Our aim is to let everyone contribute to assuring quality instead of putting all the effort on a single group.

#### Centralized & Actionable

Growing products make it challenging to keep up with the sheer amount of tests and test data generated. We built a centralised dashboard that acts as a single pane of glass for everything test related.

Our analysis tools facilitate developers and testers with troubleshooting failed tests

> and give engineering managers strategic insights into their current testing practices.

Reviewer: ```suggestion
and gives engineering managers strategic insights into their current testing practices.
```

That way everyone can make better decisions faster.

> #### Integrates within the bigger Software Testing Life Cycle

Reviewer: ```suggestion
#### Integrate within the Larger Software Testing Life Cycle
```

> Test execution is a small part of a bigger picture. Testkube integrates with the bigger ecosystem. You can run executions from external tools through triggers, events and use webhooks to keep other tools in the loop.

Reviewer: ```suggestion
Test execution is a small part of a bigger picture. Testkube integrates into the bigger ecosystem. You can run executions from external tools through triggers, events and use webhooks to keep other tools in the loop.
```

> ## How does it work in a nutshell?

Reviewer: ```suggestion
## How does it work?
```
